cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -539,32 +547,51 @@ class TLMResponse(TypedDict):
 
 class TLMOptions(TypedDict):
     """Typed dict containing advanced configuration options for the Trustworthy Language Model.
-    Many of these arguments are automatically determined by the quality preset selected
-    (see the arguments in the TLM [initialization method](../studio#method-tlm) to learn more about the various quality presets),
-    but specifying custom values here will override any default values from the quality preset.
+    Many of these configurations are automatically determined by the quality preset selected
+    (see the arguments in the TLM [initialization method](../studio#method-tlm) to learn more about quality presets).
+    Specifying custom values here will override any default values from the quality preset.
+
+    For all options described below, higher/more expensive settings will lead to longer runtimes and may consume more tokens internally.
+    The high token cost might make it such that you are not able to run long prompts (or prompts with long responses) in your account,
+    unless your token limits are increased. If you are hit token limit issues, try using lower/less expensive settings
+    to be able to run longer prompts/responses.
+
+    The default values corresponding to each quality preset (specified when instantiating [`Studio.TLM()`](../studio/#method-tlm)) are:
+    - **best:** `num_candidate_responses` = 6, `num_consistency_samples` = 8, `use_self_reflection` = True. This preset will improve LLM responses.
+    - **high:** `num_candidate_responses` = 6, `num_consistency_samples` = 8, `use_self_reflection` = True. This preset will improve LLM responses.
+    - **medium:** `num_candidate_responses` = 1, `num_consistency_samples` = 8, `use_self_reflection` = True
```

Comment:
concern: this was listed as 4.

It should be 8 right?

can  you verify medium preset is using `num_consistency_samples` = 8 not 4?

Reply:
Yes sorry it is 8, this was a typo.